src/test/resources/org/datadog/jenkins/plugins/datadog/listeners/testPipelineDefinition.txt

```diff
@@ -16,22 +18,26 @@ pipeline {
                         stage('Windows-1'){
                             steps{
                             sh "echo 'Windows 1'"
+                            exit 1
```

Comment:
These are needed because the result of this pipeline's stages are inconsistent locally and on the CI. A lot of the stages would fail on the windows images but not the linux images. this makes them fail all the time. There's probably a better way to make the result consistent such as spying that I could look into.

Reply:
Oh the "sh" step does not exist on windows right?
I think we should be able to replace all `sh "echo XX"` by simply `echo XX` which should be cross platform